python/pyarrow/table.pxi

```diff
@@ -1155,9 +1155,9 @@ cdef class Table(_PandasConvertible):
 
         Parameters
         ----------
-        arrays: list of pyarrow.Array or pyarrow.Column
+        arrays : list of pyarrow.Array or pyarrow.Column
```

Comment:
This is actually not needed in the latest numpydoc spec. But for docs improvement, we could probably build one day on pandas' work: https://github.com/pandas-dev/pandas/pull/22408